src/main/java/com/microsoft/azure/functions/annotation/CosmosDBTrigger.java

```diff
@@ -21,18 +21,20 @@
  *     <li>Any POJO type</li>
  * </ul>
  *
- * <p>The following example shows a cosmos db trigger which logs the count of the returned items:</p>
+ * <p>The following example shows a Java function that is invoked when there are inserts or updates in the specified
+ * database and collection.</p>
  *
- * <pre>{@literal @}FunctionName("cdbprocessor")
- * public void cosmosDbProcessor(
- *    {@literal @}CosmosDBTrigger(name = "items",
- *                      databaseName = "mydbname",
- *                      collectionName = "mycollname",
- *                      leaseCollectionName = "",
- *                      connectionStringSetting = "myconnvarname") MyDataItem[] items,
+ * <pre>{@literal @}FunctionName("cosmosDBMonitor")
+ * public void cosmosDbLog(
+ *    {@literal @}CosmosDBTrigger(name = "database",
+ *                      databaseName = "ToDoList",
+ *                      collectionName = "Items",
+ *                      leaseCollectionName = "leases",
+ *                      createLeaseCollectionIfNotExists = true,
+ *                      connectionStringSetting = "AzureCosmosDBConnection") String[] items,
```

Comment:
Input as String[]  does this work? What is the sample data you use to trigger this function?

Reply:
I just guessed the type. That's why we need to confirm with the latest bits.